### PR TITLE
Done

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lib/numbers.md [hunk 1]
```diff
@@ -5,20 +5,23 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+When working with numbers '+' will add two numbers together and return thier sum.  When working with strings, '+' will add two strings togeter - i.e. "Hello" + "World" would
+return "HelloWorld".
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, it would not work.  In order for a mathematical operation to work on a string, the string must first be transformed into an integer using '.to_i'.
```
Comment (sumeetjain) [inline lib/numbers.md:22]: How about if I had a string `"30.75"` and wanted to divide it by 3? What would happen if you used `to_i` in this case?

File: lib/numbers.md [hunk 2]
```diff
@@ -5,20 +5,23 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
 
 # What is the purpose of the `times` operation? Is that the same as `*`?
 
-Replace me with your response...
+The '.times'method is not the same as '*'.  While '*' can be used to multiply strings, it will return the result as one long string.  The '.times' method will execute
+a command x number of times and will return the result each as its own string.
```
Comment (sumeetjain) [inline lib/numbers.md]: This is correct:

> The '.times' method will execute a command x number of times

But this is unclear:

> and will return the result each as its own string

Is the result of the `times` operation always a string? What are some examples of Ruby code using `times`?

Also, how would you write Ruby code to accomplish the following task: Create a variable `account_balance` and set it to the number `0`. Now execute a block of code 5 times, where each time it adds `5` to `account_balance`.

File: lib/numbers.md (hunk 1 above)
Comment (sumeetjain) [inline lib/numbers.md:22]: Just making sure you saw the above comment. I can approve this assignment once you address that question.

**Comment by ghost** [comment]
.to_i will return only integers, and I will round down to the nearest full number which in this case is 30.  If we wanted to divide "30.75" by 3 I would need to convert it into a float first using .to_f.  This preserves the decimal value so that when I divide by 3 Ruby will return "10.25"

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**